pom.xml

```diff
@@ -106,18 +106,33 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>script-security</artifactId>
-      <version>1313.v7a_6067dc7087</version>
+      <version>1341.va_2819b_414686</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-step-api</artifactId>
-      <version>639.v6eca_cd8c04a_a_</version>
+      <version>678.v3ee58b_469476</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-cps</artifactId>
       <version>3837.v305192405b_c0</version>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-support</artifactId>
+      <version>848.v5a_383b_d14921</version>
+    </dependency>
```

Comment:
I recommend not pinning dependencies of plugins that have the `org.jenkins-ci.plugins` groupId. The tested/validated version pairings are set in the Jenkins plugin Bill of Materials (BOM). See https://github.com/jenkinsci/bom

Reply:
@adambkaplan Removed pinned dependencies of plugins as per suggestion. Please review. Thanks